### ISSUE TITLE
Process: maintainer involvement in triaging issues

### ISSUE THREAD

**Issue by mbolivar-nordic** [zephyrproject-rtos/zephyr#48630]
This is a re-hash of the initial discussion in #46759.

Following some offline discussion with @nashif, we agreed that it would be worthwhile to start sending requests for action from maintainers to periodically triage open bugs and issues in parallel to the ongoing broader discussions about release criteria that turned into https://github.com/zephyrproject-rtos/zephyr/issues/38566.

To resolve in particular:

- Triaging of issues, especially low priority issues: what to use? "known issue" label? "milestone" metadata?
- Any other triaging for initial effort?

**Comment by mbolivar-nordic** [comment]
Process WG:

For the initial triage, first experiment, not limiting future work to this:

- Ask maintainers to use 'future' milestone for bugs they likely won't be able to address this release
- We (the TSC) will exclude these low priority bugs from our release metrics
- We (the release managers) will clear the 'future' milestone from every bug after each release, to enforce policy that we revisit this
- Preserve 50 low priority bugs (without future milestone) as release criterion target for v3.2

AIs:

- @mbolivar-nordic update my scripts to generate these emails
- @stephanosio to work in the long term about setting a bot up
- @mbolivar-nordic to send the initial emails

**Comment by mbolivar-nordic** [comment]
Process WG: moved to done; TSC has approved
